rules/php73/src/Rector/FuncCall/JsonThrowOnErrorRector.php

```diff
@@ -98,11 +99,27 @@ private function processJsonDecode(FuncCall $funcCall): ?FuncCall
 
         $funcCall->args[3] = new Arg($this->createConstFetch('JSON_THROW_ON_ERROR'));
 
+        $this->tryToConvertErrorHandling($funcCall);
+
         return $funcCall;
     }
 
     private function createConstFetch(string $name): ConstFetch
     {
         return new ConstFetch(new Name($name));
     }
+
+    private function tryToConvertErrorHandling(FuncCall $funcCall): void
+    {
+        $condition = $funcCall->getAttribute(AttributeKey::NEXT_NODE); // Gives NULL
+        if (! $condition instanceof Node\Stmt\If_) {
```

Comment:
When running `convert_conditional_error_check_to_catch.php.inc` test, I expect here to get the `if` node, but it gives `NULL`.

Could you give a hint on how to get a node?

Reply:
If you dump() on `$funcCall`, is there something you can use?

Reply:
parent node (function)

Reply:
Does it solve the issue?

Reply:
You mean I should manually iterate over statements in a function, find the `json_encode` one and then look for the `if` condition after it?

Reply:
I think in this case there is no other way to do it. 

There are also other related attibutes, but I don't think they will help:

- AttributeKey::PREVIOUS_STATEMENT
- AttributeKey::CURRENT_STATEMENT